File: Examen/C/MayorYMenor.c
#include <stdio.h>

int main() {
    int n, i, num, mayor, menor;

    printf("Ingrese la cantidad de numeros: ");
    scanf("%d", &n);

    printf("Ingrese el numero 1: ");
    scanf("%d", &num);
    mayor = num;
    menor = num;

    for(i = 2; i <= n; i++) {
        printf("Ingrese el numero %d: ", i);
        scanf("%d", &num);

        if(num > mayor) mayor = num;
        if(num < menor) menor = num;
    }

    printf("El numero mayor es: %d\n", mayor);
    printf("El numero menor es: %d\n", menor);

    return 0;
}

File: Examen/Fortran/MayorYMenor.f
      PROGRAM MayorMenor
      INTEGER N, I, NUM
      INTEGER MAYOR, MENOR

      PRINT *, 'Ingrese la cantidad de numeros:'
      READ *, N

      PRINT *, 'Ingrese el numero 1:'
      READ *, NUM
      MAYOR = NUM
      MENOR = NUM

      DO 10 I = 2, N
         PRINT *, 'Ingrese el numero ', I, ':'
         READ *, NUM
         IF (NUM > MAYOR) MAYOR = NUM
         IF (NUM < MENOR) MENOR = NUM
10    CONTINUE

      PRINT *, 'El numero mayor es: ', MAYOR
      PRINT *, 'El numero menor es: ', MENOR

      PRINT *, 'Presione ENTER para salir...'
      READ(*,*)
      END

